docs/source/compiler.rst

```diff
@@ -113,6 +113,30 @@ the previous example snippet is identical to the following:
     ep = qc.compiler.native_quil_to_executable(np)
     print(ep.program) # here ep is of type PyquilExecutableResponse, which is not always inspectable
 
+Timeouts
+--------
+
+If your circuit is sufficiently complex the compiler may require more time than is permitted by
+default (``10`` seconds). To relax this timeout, you may use the `set_timeout` method:
+
+.. code:: python
+
+    qc = get_qc(...)
+    qc.compiler.set_timeout(100) # 100 seconds
+
+Prior to version 2.25, process was slightly different. For a QVM
```

Comment:
This seems more like a description of this PR rather than something in documentation. I don't know if there's value in explaining how to use an old version in new documentation. If you want, a `(New in 2.25)` would do it, but I haven't seen that pattern elsewhere in our docs.

Reply:
I agree less is more here. Maybe just mention the change is fully backwards compatible.